README.md

```diff
@@ -77,28 +77,28 @@ To run with the example plugins, do the following:
         - example-blockstore:example-default
 ```
 
-5. Run `kubectl create -f ark-blockstore-example/with-pv.yaml` to apply a sample nginx application that uses the example block store plugin. ***Note***: This example works best on a virtual machine, as it uses the host's `/tmp` directory for data storage.
+5. Run `kubectl create -f velero-blockstore-example/with-pv.yaml` to apply a sample nginx application that uses the example block store plugin. ***Note***: This example works best on a virtual machine, as it uses the host's `/tmp` directory for data storage.
```

Comment:
this file doesn't exist (though neither does the ark one) -- I think this should be `examples/with-pv.yaml`?